docs/src/running-tests-js.md

```diff
@@ -2,112 +2,145 @@
 
-  ```bash
-  npx playwright test
-  ```
+```bash
+npx playwright test --ui
+```
+
+![UI Mode](https://github.com/microsoft/playwright/assets/13063165/c5b501cc-4f5d-485a-87cc-66044c651786)
 
-- Running a single test file
+Check out or [detailed guide on UI Mode](./test-ui-mode.md) to learn more about it's features.
 
-  ```bash
-  npx playwright test landing-page.spec.ts
-  ```
+### Run tests in headed mode
 
-- Run a set of test files
+To run your tests in headed mode use the `--headed` flag. This will open up a browser window while running your tests and once finished the browser window will close.
+
+```bash
+npx playwright test --headed
+```
+
+### Run tests on different browsers
+
+To specify which browser you would like to run your tests on use the `--project` flag followed by the name of the browser.
+
+```bash
+npx playwright test --project webkit
+```
+
+To specify multiple browsers to run your tests on use the `--project` flag multiple times followed by the name of each browser.
+
+```bash
+npx playwright test --project webkit --project firefox
+```
 
-  ```bash
-  npx playwright test tests/todo-page/ tests/landing-page/
-  ```
+### Run specific tests
 
-- Run files that have `landing` or `login` in the file name
+To run a single test file pass in the name of the test file that you want to run.
 
-  ```bash
-  npx playwright test landing login
-  ```
+```bash
+npx playwright test landing-page.spec.ts
+```
 
-- Run the test with the title
+To run a set of test files from different directories pass in the names of the directories that you want to run the tests in.
 
-  ```bash
-  npx playwright test -g "add a todo item"
-  ```
+```bash
+npx playwright test tests/todo-page/ tests/landing-page/
+```
 
-- Running tests in headed mode
+To run files that have `landing` or `login` in the file name simply pass in these keywords to the CLI.
 
-  ```bash
-  npx playwright test landing-page.spec.ts --headed
-  ```
+```bash
+npx playwright test landing login
+```
 
-- Running tests on a specific project
+To run a test with a specific title use the `-g` flag followed by the title of the test.
 
-  ```bash
-  npx playwright test landing-page.ts --project=chromium
-  ```
+```bash
+npx playwright test -g "add a todo item"
+```
 
-## Debugging Tests
+## Debugging tests
 
 Since Playwright runs in Node.js, you can debug it with your debugger of choice e.g. using `console.log` or inside your IDE or directly in VS Code with the [VS Code Extension](./getting-started-vscode.md). Playwright comes with the [Playwright Inspector](./debug.md#playwright-inspector) which allows you to step through Playwright API calls, see their debug logs and explore [locators](./locators.md).
 
 
-- Debugging all tests:
+To debug all tests run the Playwright test command followed by the `--debug` flag.
```

Comment:
We should add a note here that we recommend UI Mode for debugging. --debug and page.pause() are last resort. And only if thats not enough, we recommend --debug.